### PR TITLE
Adjust operators' classification and xref them

### PR DESCRIPTION
**Pull request by stoned**

### CODE REVIEW

File: doc/Type/Callable.pod6 [hunk 1]
```diff
@@ -30,7 +30,8 @@ of such a container is C<Callable>.
 
     method CALL-ME(Callable:D $self: |arguments)
 
-This method is required for postfix:«( )» and postfix:«.( )». It's what makes
+This method is required for L«C<( )> postcircumfix operator|/language/operators#postcircumfix_(_)»
```
Comment (JJ) [inline doc/Type/Callable.pod6:33]: As long as we're modifying this, maybe add "the"?

File: doc/Type/Callable.pod6 [hunk 2]
```diff
@@ -30,7 +30,8 @@ of such a container is C<Callable>.
...
+and L«C<.( )> postcircumfix operator|/language/operators#index-entry-.(_)». It's what makes
```
Comment (JJ) [inline doc/Type/Callable.pod6:34]: Ditto

**Review by JJ — APPROVED** [review]
Looks fine, maybe consider the comments. Thanks!

File: doc/Type/Callable.pod6 (hunk 1 above)
Comment (stoned) [reply doc/Type/Callable.pod6]: Sure ! Done.

File: doc/Type/Callable.pod6 (hunk 2 above)
Comment (stoned) [reply doc/Type/Callable.pod6]: ... and done.

**Pull request merged**